etc/systemd/system/anabot.service

```diff
@@ -12,7 +12,7 @@ ConditionArchitecture=!s390x
 # It seems the delay shouldn't be needed with Wayland (and/or with the new version
 # of dogtail). If it turns out it's not the case, we should remove the sleep
 # from here and implement it it launcher.py.
-ExecStartPre=sh -c "ps aux | grep -q gnome-kiosk || sleep 30"
+ExecStartPre=sh -c "ps aux | grep -q Xorg && sleep 30"
```

Comment:
Actually, this might still be problematic, for example when an application is executed with a possible `--no-Xorg` argument.
What about using `which Xorg` instead of listing running processes?

Reply:
Thanks for having a look, that's actually a good point. Fixed.